### ISSUE TITLE
make:entity class name got trimmed by D

### ISSUE THREAD

**Issue by chapterjason** [symfony/maker-bundle#148]
While I worked on a project I found the following. The class name got trimmed by the `D`.

```
 Add another property? Enter the property name (or press <return> to stop adding fields):
 > directory

 Field type (enter ? to see all types) [string]:
 > relation

 What class should this entity be related to?:
 > Directory

What type of relationship is this?
 ------------ --------------------------------------------------------------- 
  Type         Description                                                    
 ------------ --------------------------------------------------------------- 
  ManyToOne    Each File relates to (has) one irectory.                       
               Each irectory can relate/has to (have) many File objects       
                                                                              
  OneToMany    Each File relates can relate to (have) many irectory objects.  
               Each irectory relates to (has) one File                        
                                                                              
  ManyToMany   Each File relates can relate to (have) many irectory objects.  
               Each irectory can also relate to (have) many File objects      
                                                                              
  OneToOne     Each File relates to (has) exactly one irectory.               
               Each irectory also relates to (has) exactly one File.          
 ------------ --------------------------------------------------------------- 

 Relation type? [ManyToOne, OneToMany, ManyToMany, OneToOne]:
 > ManyToOne

 Is the File.directory property allowed to be null (nullable)? (yes/no) [yes]:
 > no

 Do you want to add a new property to irectory so that you can access/update File objects from it - e.g. $irectory->getFiles()? (yes/no) [yes]:
 > 

 A new property will also be added to the irectory class so that you can access the related File objects from it.

 New field name inside irectory [files]:
 > 

 Do you want to activate orphanRemoval on your relationship?
 A File is "orphaned" when it is removed from its related irectory.
 e.g. $irectory->removeFile($file)
 
 NOTE: If a File may *change* from one irectory to another, answer "no".

 Do you want to automatically delete orphaned App\Entity\File objects (orphanRemoval)? (yes/no) [no]:
 > yes


In ClassSourceManipulator.php line 662:
                                 
  Could not find namespace node  
                                 

make:entity [--regenerate] [--overwrite] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command> [<name>]


```

**Comment by weaverryan** [comment]
Wow! This is totally weird! Not only does the “d” become missing suddenly in the messages, but it erroree at the end! What entity were you generating this into? Can you show the source?

**Comment by chapterjason** [comment]
Sure

File
```
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use Gedmo\Mapping\Annotation as Gedmo;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * @ORM\Entity(repositoryClass="App\Repository\FileRepository")
 */
class File
{
    /**
     * @ORM\Id()
     * @ORM\GeneratedValue()
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $title;

    /**
     * @Gedmo\Slug(fields={"title"})
     * @ORM\Column(length=128, unique=true)
     */
    private $slug;

    /**
     * @ORM\Column(type="string")
     * @Assert\Image()
     */
    private $file;

    /**
     * @ORM\ManyToOne(targetEntity="App\Entity\Directory", inversedBy="files")
     * @ORM\JoinColumn(nullable=false)
     */
    private $directory;

    public function getId()
    {
        return $this->id;
    }

    public function getTitle(): ?string
    {
        return $this->title;
    }

    public function setTitle(string $title): self
    {
        $this->title = $title;

        return $this;
    }

    public function getSlug(): ?string
    {
        return $this->slug;
    }

    public function setSlug(string $slug): self
    {
        $this->slug = $slug;

        return $this;
    }

    public function getFile(): ?string
    {
        return $this->file;
    }

    public function setFile(string $file): self
    {
        $this->file = $file;

        return $this;
    }

    public function getDirectory(): ?Directory
    {
        return $this->directory;
    }

    public function setDirectory(?Directory $directory): self
    {
        $this->directory = $directory;

        return $this;
    }
}
```

and Directory
```
<?php

namespace App\Entity;

use Doctrine\Common\Collections\ArrayCollection;
use Doctrine\Common\Collections\Collection;
use Doctrine\ORM\Mapping as ORM;
use Gedmo\Mapping\Annotation as Gedmo;

/**
 * @ORM\Entity(repositoryClass="App\Repository\DirectoryRepository")
 */
class Directory
{
    /**
     * @ORM\Id()
     * @ORM\GeneratedValue()
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $title;

    /**
     * @Gedmo\Slug(fields={"title"})
     * @ORM\Column(length=128, unique=true)
     */
    private $slug;

    /**
     * @ORM\OneToMany(targetEntity="App\Entity\File", mappedBy="directory", orphanRemoval=true)
     */
    private $files;

    /**
     * @ORM\OneToMany(targetEntity="App\Entity\Directory", mappedBy="parent")
     */
    private $children;

    /**
     * @ORM\ManyToOne(targetEntity="App\Entity\Directory", inversedBy="children")
     * @ORM\JoinColumn(name="parent_id", referencedColumnName="id")
     */
    private $parent;

    public function __construct()
    {
        $this->children = new ArrayCollection();
        $this->files = new ArrayCollection();
    }

    public function getId()
    {
        return $this->id;
    }

    public function getTitle(): ?string
    {
        return $this->title;
    }

    public function setTitle(string $title): self
    {
        $this->title = $title;

        return $this;
    }

    public function getSlug(): ?string
    {
        return $this->slug;
    }

    public function setSlug(string $slug): self
    {
        $this->slug = $slug;

        return $this;
    }

    /**
     * @return Collection|Directory[]
     */
    public function getChildren(): Collection
    {
        return $this->children;
    }

    public function addChildren(Directory $children): self
    {
        if (!$this->children->contains($children)) {
            $this->children[] = $children;
            $children->setParent($this);
        }

        return $this;
    }

    public function removeChildren(Directory $children): self
    {
        if ($this->children->contains($children)) {
            $this->children->removeElement($children);
            // set the owning side to null (unless already changed)
            if ($children->getParent() === $this) {
                $children->setParent(null);
            }
        }

        return $this;
    }

    /**
     * @return Directory|null
     */
    public function getParent(): ?Directory
    {
        return $this->parent;
    }

    /**
     * @param Directory|null $parent
     * @return Directory
     */
    public function setParent(?Directory $parent): self
    {
        $this->parent = $parent;

        return $this;
    }

    /**
     * @return Collection|File[]
     */
    public function getFiles(): Collection
    {
        return $this->files;
    }

    public function addFile(File $file): self
    {
        if (!$this->files->contains($file)) {
            $this->files[] = $file;
            $file->setDirectory($this);
        }

        return $this;
    }

    public function removeFile(File $file): self
    {
        if ($this->files->contains($file)) {
            $this->files->removeElement($file);
            // set the owning side to null (unless already changed)
            if ($file->getDirectory() === $this) {
                $file->setDirectory(null);
            }
        }

        return $this;
    }
}

```

**Comment by weaverryan** [comment]
Thanks for the extra info! So, this should be fixed in #150 (and I'll make a new release shortly). If you still see any issues once the release is available, please let us know!
